contracts/basset_vault/src/contract.rs

```diff
@@ -69,6 +71,8 @@ pub fn instantiate(
         manual_ltv: msg.manual_ltv,
         fee_rate: msg.fee_rate,
         tax_rate: msg.tax_rate,
+        nasset_psi_swap_contract_addr: "".to_string(),
```

Comment:
Mmm, I think better to do something like this: https://github.com/terraswap/terraswap/blob/97cefa337798bdb0cba0327dd4152607839a5c77/contracts/terraswap_factory/src/state.rs#L24

Reply:
Or... maybe not. Forget about it.